src/main/java/uk/gov/ons/ssdc/caseprocessor/schedule/FulfilmentScheduler.java

```diff
@@ -34,9 +34,10 @@ public FulfilmentScheduler(
   @Scheduled(fixedDelayString = "${scheduler.frequency}")
   @Transactional
   public void scheduleFulfilments() {
-    if (!clusterLeaderManager.isThisHostClusterLeader()) {
-      return; // This host (i.e. pod) is not the leader... don't do any scheduling
-    }
+    //    Keeps going wrong debugging, thinks it's not the leader
```

Comment:
Any idea what's going on with this? Bit concerning that it seems to break so easily.

Reply:
I think it was related to my running my docker-dev case-proc and maybe running the debugger at the same time.  Then when I stopped the docker-dev case-proc, the debugger version hadn't yet cycled into clusterLeader.
I may have just been impatient a few times,  but got fed up with waiting on this.

I'm still not a fan on this pattern,  I think we discussed the dreaded UPDATED FOR SKIPPED LOCKED yonks ago - which seems far less complicated.